server/lib/transforms/html/table.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const match = require('@quarterto/cheerio-match-multiple');
+
+function generateTableHref(el) {
+	const tableId = el.attr('id');
+	const contentId = tableId.match(/table-(.*)_\d/)[1];
```

Comment:
Are you sure there is no other way to get the content ID from the content information received via ES-interface? If we could do that we could avoid the dependency to the name pattern applied to the ID.
In this way it's really fragile.